commit d0a4e11a2edc7f4e66552d8753c92833de087ab7
Author: K. Y. Srinivasan <kys@microsoft.com>
Date:   Thu Dec 22 16:54:01 2016 -0800

    Drivers: hv: util: kvp: Fix a rescind processing issue
    
    commit 5a66fecbf6aa528e375cbebccb1061cc58d80c84 upstream.
    
    KVP may use a char device to support the communication between
    the user level daemon and the driver. When the KVP channel is rescinded
    we need to make sure that the char device is fully cleaned up before
    we can process a new KVP offer from the host. Implement this logic.
    
    Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1321_d0a4e11a2edc7f4e66552d8753c92833de087ab7.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1321_d0a4e11a2edc7f4e66552d8753c92833de087ab7.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
